scripts/h52csv.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python3
+
+import numpy as np
+import pandas as pd
+import h5py
+
+import sys
+from pathlib import Path
+
+path = Path(sys.argv[1])
+df = h5py.File(path, 'r')
```

Comment:
```suggestion
file = h5py.File(path, 'r')
```
The name `df` might be misleading since it does not hold a Pandas DataFrame object.